### ISSUE TITLE
GUI : FED : Firmware Access Key

### ISSUE THREAD

**Issue by derick-montague** [ibm-openbmc/dev#1290]
## Summary
Add access key expiration date and ability to save new access key.

### UX Flow / Interaction Requirements
- [x] Displays access key expiration date on Firmware page
- [x] Add edit icon. Clicking edit icon will enable user to enter new access key (access key validation - must be 34 characters)
- [x] Create the Access key modal
- [ ] Add save in FirmwareStore.js

### cURL Commands and Redfish Response
- [x] Get Access key date from `https://${bmc}/redfish/v1/LicenseService/Licenses/UAK` - `ExpirationDate`
- [ ] `curl -k -H "Content-Type: application/json" -X
POST https://${bmc}/redfish/v1/LicenseService/Licenses -d '{"LicenseString": "<34-character-string-nospaces>"}'`
- [x] Max length = 34
- [x] Should include title for icon only button

### Notes
- [x] Consider adding info tooltip if needed
- [ ] Consider adding status icons (warning/danger) when nearing expiration/expired

#### Browser Tests (Chrome, Firefox, Safari (Mac), Edge (Windows))
- [x] Tested responsive layout on supported viewport sizes
- [x] Verified functionality works as expected, e.g. success and failure use cases
- [x] Resolved any JavaScript errors thrown to the console

#### Accessibility Tests
- [x] Verified markup is semantic
- [x] No reported issues in IBM Accessibility Checker (Free open source tool) https://www.ibm.com/able/toolkit/tools/

#### Test Hooks
- [ ] Verify need for `data-test-id` hooks with test team

### Screenshots
![Screen Shot 2021-02-03 at 1 10 33 PM](https://user-images.githubusercontent.com/8097306/106810212-7d797d80-6621-11eb-870c-a5b6ee160198.png)
![Screen Shot 2021-02-03 at 1 10 45 PM](https://user-images.githubusercontent.com/8097306/106810235-84a08b80-6621-11eb-8838-b2b8c2cc58bf.png)

**Comment by nicoleconser** [comment]
Suggest renaming this story. The mock-ups above are referring to **Firmware Access Key**, which is completely separate from the **Firmware License Agreement**.

**Comment by derick-montague** [comment]
- Firmware access key milestone updated in feature spreadsheet also. 
- Watch the dependent stories for when this story can be implemented. It may still be possible in the A.1.211 milestone.

**Comment by derick-montague** [comment]
Milestone update based on [October feature sync meeting](https://ibm.box.com/s/g5jveq2vo3dba6chljjzvkcq7b58646u).

**Comment by derick-montague** [comment]
This will be part of a licensing service. It has been suggested by engineering that this should be part of #1538. I have added a note about the licensing server and when this requirement was added in t[his note](https://github.com/ibm-openbmc/dev/issues/1538#issuecomment-783448321). I have concerns about how this could potentially change the FW design. It may have minimal impact.

However, for the concern noted above, I am making this story dependent on CoD.
